easybuild/easyconfigs/g/Globus-CLI/Globus-CLI-3.34.0-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,42 @@
+# This easyconfig was created by Simon Branford of the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'Globus-CLI'
+version = '3.34.0'
+
+homepage = "https://docs.globus.org/cli/"
+description = """A Command Line Wrapper over the Globus SDK for Python, which provides an interface to Globus services
+ from the shell, and is suited to both interactive and simple scripting use cases."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+builddependencies = [
+    ('binutils', '2.42'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Python-bundle-PyPI', '2024.06'),
+]
+
+exts_list = [
+    ('jmespath', '1.0.1', {
+        'checksums': ['90261b206d6defd58fdd5e85f478bf633a2901798906be2ad389150c5c60edbe'],
+    }),
+    ('pyjwt', '2.10.1', {
```

Comment:
```suggestion
    ('PyJWT', '2.10.1', {
```

Reply:
but the name on pypi is still `PyJWT`, so i'd keep it like that
https://pypi.org/project/PyJWT/

Reply:
ooooooooooooooooooooooook done in https://github.com/easybuilders/easybuild-easyconfigs/pull/23025/commits/3dbdce5d6b8aa166a40a03547c3069380dfa25cd :slightly_smiling_face: